### DISCUSSION TITLE
Сделайте фронт на моем спа фраемворке - https://github.com/ru51a4/minimal_react

### DISCUSSION THREAD

**Discussion by ru51a4** [OpenVK/openvk#1084] [Ideas]
Сделайте фронт на моем спа фраемворке - https://github.com/ru51a4/minimal_react

**Comment by NoPlagiarism** [comment]
... То есть всю пыху переписать на реактоподобный NN фреймворк?...

**Reply by ru51a4** [reply]
сделайте просто еще один веб клиент

**Reply by NoPlagiarism** [reply]
> сделайте просто еще один веб клиент

Это невозможно полноценно сделать, если брать в учёт как устроен OVK

Не, ну спасибо, что хоть issues подобными реквестами не засираете 👍

**Reply by ru51a4** [reply]
можно сделать вставки minimal_react в вьюшки php, например сделать чат на сокетах
а моб приложение разве не работает?

**Reply by ru51a4** [reply]
поддержите отечественное по 🇷🇺👍

**Reply by veselcraft** [reply]
пиздец

**Reply by NoPlagiarism** [reply]
> поддержите отечественное по 🇷🇺👍

Поставь Astra Linux на собственный ПК как основную операционку. Поддержи отечественное ПО и плевать насколько это будет удобно и уместно

Может ещё Эльбрус, который в основном для всяких серваков, себе в геймерский корпус встроишь? Тоже поддержка соотечественников
